### PR TITLE
Update to dor-services 8

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

So this code uses the same version of dor-services used by other applications

## Was the usage documentation (e.g. README, consul, DevOpsDocs, wiki) updated?
n/a

### CODE REVIEW

File: config/environments/test.rb [hunk 1]
```diff
@@ -1,11 +1,2 @@
 Dor::Config.configure do
```
Comment (ndushay) [inline config/environments/test.rb]: we still need Dor::Config even tho it's empty?  if so, please add a comment.  if not, remove this file?

File: config/settings.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+dor_services:
+  url: 'https://test'
+  token: 'test'
+
+sleep_time: 0
```
Comment (ndushay) [inline config/settings.yml]: comments explaining when sleep_time and num_attempts come into play wouldn't go amiss here, just as you have them in shared_configs.

File: config/environments/test.rb (hunk 1 above)
Comment (jcoyne) [reply config/environments/test.rb]: We need to have the file here. I thought I'd leave the config in here as this is the sort of thing in shared_configs version of the file.

Comment (ndushay) [reply config/environments/test.rb]: ok, cool.   Can you include a comment tho?  ("we need this file, even if it's empty, b/c ...")

**Comment by jcoyne** [comment]
Shared configs PRs have been merged, just waiting for +1 on this so I can do the deployment.

**Pull request merged**